packages/infrastructure/kube_airbyte/README.md

```diff
@@ -0,0 +1,28 @@
+# Airbyte
+
+This module deploys Airbyte onto a Kubernetes cluster with a focus on AWS infrastructure, though it can be adapted for other cloud providers.
+
+## Scope and Connectors
+
+This module only deploys the core Airbyte engine components required for the platform to function. It does not include or configure any source or destination connectors, which must be installed separately after deployment. The Airbyte platform provides a connector catalog within its user interface where administrators can install the specific connectors needed for their data integration workflows.
+
+To install connectors:
+
+1. After deployment, log in to the Airbyte UI using the credentials provided
+2. Navigate to the "Sources" or "Destinations" section
+3. Search for and install the required connectors from the catalog
+
+For custom connector development, this module includes the Connector Builder Server component, which provides a development environment for creating and testing custom connectors to meet specialized integration needs.
+
+If you need to pre-install specific connectors or automate connector configuration, consider implementing additional Terraform modules that interact with the Airbyte API after core deployment is complete.
+
```

Comment:
Needs to have documentation for how to give airbyte additional permissions to access other AWS resources for connectors. In other words, how to grant more permissions to the service account.